activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -575,9 +573,17 @@ def build_insert_sql(insert) # :nodoc:
         "INSERT #{insert.into} #{insert.values_list}"
       end
 
+      def get_database_version
+      end
+
+      def database_version
+        schema_cache.database_version
+      end
+
+      def check_version
```

Comment:
Let's mark these "private" 

```suggestion
      def check_version # :nodoc:
```